File: 06-eggcartons-Java/eggcartons.java
// # Write the function eggCartons(eggs) that takes 
// # a non-negative integer number of eggs, and returns 
// # the smallest integer number of cartons required to hold 
// # that many eggs, where a carton may hold up to 12 eggs

class eggcartons {
	public int fun_eggcartons(int eggs){
		int n;
		while(eggs>0){
			if(eggs<12) return 1;
			n = 1 + eggs/12;
		}
		return n;
	}
}

File: 06-eggcartons-Python/eggcartons.py
# Write the function eggCartons(eggs) that takes 
# a non-negative integer number of eggs, and returns 
# the smallest integer number of cartons required to hold 
# that many eggs, where a carton may hold up to 12 eggs


def fun_eggcartons(eggs):
	# your code goes here
	if eggs>0 :
		a = eggs%12
		if a == 0 : return eggs//12
		return eggs//12 +1
	return 0